docs/jwst/pipeline/main.rst

```diff
@@ -39,60 +39,35 @@ observing modes, including:
 
 - :ref:`calwebb_dark <calwebb_dark>` for processing dark exposures
 - :ref:`calwebb_guider <calwebb_guider>` for calibrating FGS guide star data
-- :ref:`calwebb_wfs-image2 <calwebb_wfs-image2>` for stage 2 WFS&C processing
```

Comment:
Oh, OK, I get it now. There's no longer any need for `calwebb_wfs-image2`, because the step skipping is handled by the `pars-image2pipeline` ref file, in the same way there's no need for `calwebb_tso1` or `calwebb_tso-spec2` anymore. Never mind.